packages/core/core/src/services/document-service/repository.ts

```diff
@@ -186,19 +193,60 @@ export const createContentTypeRepository: RepositoryFactoryMethod = (
       populate: getDeepPopulate(uid, { relationalFields: ['id'] }),
     });
 
-    const clonedEntries = await async.map(
-      entriesToClone,
-      async.pipe(
-        validateParams,
-        omit(['id', 'createdAt', 'updatedAt']),
-        // assign new documentId
-        assoc('documentId', createDocumentId()),
-        // Merge new data into it
-        (data) => merge(data, queryParams.data),
-        (data) => entries.create({ ...queryParams, data, status: 'draft' })
-      )
-    );
+    const clonedEntries = await async.map(entriesToClone, async (entryToClone: any) => {
+      // Step 1: Create new entries with standardized original data
+      const newData = await standardizeRelations(
+        {
+          ...omit(['id', 'createdAt', 'updatedAt'], entryToClone),
+          documentId: createDocumentId(),
+        },
+        uid
+      );
+
+      const newEntry = await entries.create({
+        ...queryParams,
+        data: newData,
+        status: 'draft',
+      });
+
+      // Step 2 (optional): Update with queryParams.data if it exists, after some transformations
```

Comment:
I think we should explain in a more abstract way whats happening here, what is "standarizing" relations, and